hotnews/index.html

```diff
@@ -1,37 +1,107 @@
           }
         }
       }
-    </script>
-  </head>
-  <body>
-    <div class="flex max-w-5xl mx-auto px-4 py-1.5 text-sm font-medium">
-      <div class="flex grow items-center">
-        <img src="https://hotnews.ro/static/assets/icons/ic_updated.svg" alt="updated_icon">
-        <div class="text-hotblue">Actualizat acum un minut</div>
-      </div>
-      <div class="flex shrink">
-        <div class="text-hotblue pr-1">EUR</div>
-        <div class="text-hotgreen pr-1">4.9395</div>
-        <div class="text-hotblue pr-1">USD</div>
-        <div class="text-hotred">4.8195</div>
-      </div>
+    }
+  </script>
+</head>
+
+<body>
+  <div class="flex max-w-5xl mx-auto px-4 py-1.5 text-sm font-medium text-[14px]">
+    <div class="flex grow items-center">
+      <img src="https://hotnews.ro/static/assets/icons/ic_updated.svg" alt="updated_icon">
+      <div class="text-hotblue">Actualizat acum un minut</div>
     </div>
-    <div class='bg-hotblue h-10'>
+    <div class="flex shrink text-[14px]">
+      <div class="text-hotblue pr-1">EUR</div>
+      <div class="text-hotred pr-1">4.9395</div>
+      <div class="text-hotblue pr-1">USD</div>
+      <div class="text-hotgreen">4.8195</div>
     </div>
-  </body>
+  </div>
+  <div class="bg-hotblue">
+    <div class='mx-auto max-w-5xl h-[90px] flex'>
+      <div class="items-center justify-between flex grow">
+        <a href="../hotnews/index.html">
+          <img class="h-[52px] cursor-pointer" src="https://www.hotnews.ro/static/assets/logos/hotnews_header.svg"
+            alt="hotnews-logo"></a>
+        <div class="flex shrink">
+          <img class="m-2.5 h-[22px]" src="https://www.hotnews.ro/static/assets/icons/ic_search.svg" alt="Search-Icon">
+          <img class="m-2.5 h-[22px]" src="https://www.hotnews.ro/static/assets/icons/ic_user.svg" alt="User-Icon">
+          <img class="m-2.5 h-[22px]" src="https://www.hotnews.ro/static/assets/icons/ic_menu.svg" alt="Burger-Menu">
+        </div>
+      </div>
+    </div>
+  </div>
+  <section
+    class="p-[22px] text-hotblack mx-auto text-[18px] font-bold font-['Inter', 'sans-serif'] flex  h-14 items-center max-w-5xl">
+    <a class="pr-5 text-hotHome" href="https://www.hotnews.ro/">HOME</a>
+    <a class="pr-5 hover:text-hotEconomie" href="https://economie.hotnews.ro/">ECONOMIE</a>
+    <a class="pr-5 hover:text-hotSport" href="https://sport.hotnews.ro/">SPORT</a>
+    <a class="pr-5 hover:text-hotLife" href="https://life.hotnews.ro/">LIFE</a>
+    <a class="pr-5 hover:text-hotScience" href="https://science.hotnews.ro/">SCIENCE</a>
+    <a class="pr-5 hover:text-hotRevista" href="https://revistapresei.hotnews.ro/">REVISTA PRESEI</a>
+    <a class="pr-5 hover:text-hotDosare" href="https://www.hotnews.ro/dosare_juridice">DOSARE JURIDICE</a>
+  </section>
+
+  <div class="bg-hotblue">
+    <section
+      class="p-[22px] h-[50px] flex mx-auto max-w-5xl font-['Inter', 'sans-serif'] items-center text-hotWhite text-[15px] ">
```

Comment:
**font-['Inter', 'sans-serif']** can be replaced with your defined rule font-sans